config/groovy/util.groovy

```diff
@@ -220,6 +220,36 @@ switch (cleanerArgs[0]) {
 
         break
 
+        // Makes sure a workspace has one or more modules matching user input, most useful for grabbing whole "module distros"
+        case "init":
+            // TODO: Move most this code into module.groovy, leave an error here if you use it for something else
+            if (cleanerArgs.length == 1) {
+                println "Doing a default module init - simply fetching CoreSampleGameplay without dependencies in source (same as `groovyw module get CoreSampleGameplay`)"
```

Comment:
Applied all the logging improvements, thanks for the suggestions! Those were drafts at best :)

As for `get` vs `recurse` I wanted to keep out unneeded source unless the dev wants it - really the docs could just say to use `groovyw module get|recurse X` as desired. `groovy module init` would otherwise be about the same as init iota?

In any case I'm still not sure what to really think about the default init vs just the distros